apps/snitch_core/lib/core/data/schema/stock/stock_location.ex

```diff
@@ -13,7 +13,7 @@ defmodule Snitch.Data.Schema.StockLocation do
 
   If this is set to `true` when creating a new `StockLocation`, then a
   `StockItem` entry with `0` `:count_on_hand` and this `StockLocation` is
-  created for _all currently existing variants_.
+  created for_all currently existing variants.
```

Comment:
The underscore makes the text in italics. If you don't want italics, remove both underscores.